Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py

```diff
@@ -231,7 +231,7 @@ def get_owners_command(client: Client) -> COMMAND_OUTPUT:
 def main():
     client = Client(demisto.getParam("api_access_id"),
                     demisto.getParam("api_secret_key"),
-                    demisto.getParam("tc_api_path"))
+                    demisto.getParam("tc_api_path"),)
```

Comment:
otherwise the extra comma is redundant...
```suggestion
                    demisto.getParam("tc_api_path"),
                    )
```